tableauserverclient/server/endpoint/sites_endpoint.py

```diff
@@ -103,3 +103,31 @@ def create(self, site_item):
         new_site = SiteItem.from_response(server_response.content, self.parent_srv.namespace)[0]
         logger.info('Created new site (ID: {0})'.format(new_site.id))
         return new_site
+
+    @api(version="3.5")
+    def encrypt_extracts(self, site_id):
+        if not site_id:
+            error = "Site ID undefined."
+            raise ValueError(error)
+        url = "{0}/{1}/encrypt-extracts".format(self.baseurl, site_id)
+        empty_req = RequestFactory.Empty.empty_req()
+        self.post_request(url, empty_req)
+
+    @api(version="3.5")
+    def decrypt_extracts(self, site_id):
+        if not site_id:
+            error = "Site ID undefined."
+            raise ValueError(error)
+        url = "{0}/{1}/decrypt-extracts".format(self.baseurl, site_id)
+        empty_req = RequestFactory.Empty.empty_req()
+        self.post_request(url, empty_req)
+
+    @api(version="3.5")
+    def re_encrypt_extracts(self, site_id):
```

Comment:
What's the difference from a conceptual level between encrypt and re-encrypt? Does the user need to really care about the difference or should we switch between the two based on inspection?

Reply:
Hm, good question.  They're exposed as separate commands in tabcmd, but there's not any real justification for that. I'll check with the feature team.

Reply:
encrypt will only act on extracts that are not yet encrypted. re-encrypt will rotate the key for extracts that are already encrypted and do nothing for unencryped extracts.

Reply:
@RussTheAerialist how's that explanation? I think in our other help docs they are presented as two different operations, I don't think it would help to change that model here.

Reply:
Yup, totally makes sense! I mean, maybe I don't 100% agree with their design decisions, but 100% agree that we shouldn't change the model here.